changelogs/current.yaml

```diff
@@ -116,6 +116,9 @@ minor_behavior_changes:
 - area: logging
   change: |
     changed category name for access log filter extensions to ``envoy.access_loggers.extension_filters``.
+- area: http
```

Comment:
Should this be in the `deprecated` section?

Reply:
Yes, moving.